Livecheckables/ttyrec.rb

```diff
@@ -0,0 +1,6 @@
+class Ttyrec
+  livecheck do
+    url :homepage
+    regex(/href=.*?ttyrec-v?(\d+(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    regex(/href=['"]?ttyrec-v?(\d+(?:\.\d+)+)\.t/i)
```

There's also a `Term-TtyRec-0.03.tar.gz` archive on the page and we're in the same situation as `ifstat` where the URL is relative and we have to use `href=['"]?` as a leading delimiter.